client/my-sites/email/titan-new-mailbox/index.jsx

```diff
@@ -127,6 +138,24 @@ const TitanNewMailbox = ( {
 						/>
 					</FormLabel>
 					{ hasPasswordError && <FormInputValidation text={ passwordError } isError /> }
+					{ hiddenFieldNames.includes( TITAN_PASSWORD_RESET_FIELD ) && ! showAlternateEmail && (
+						<FormSettingExplanation>
+							{ translate( 'Your password reset email is {{strong}}%(userEmail)s{{/strong}}.', {
+								args: {
+									userEmail,
+								},
+								components: {
+									strong: <strong />,
+								},
+							} ) }{ ' ' }
+							{ /* eslint-disable-next-line jsx-a11y/anchor-is-valid */ }
+							<a href="#" onClick={ showAlternateEmailField }>
+								{ translate( 'Change it', {
```

Comment:
The trailing dot is missing. Note it shouldn't be part of the link. Thinking about it more, I think we should actually merge both sentences (an [example](https://github.com/Automattic/wp-calypso/blob/fdc68e8556f182e5dec50caf68291379be3d2312/client/my-sites/email/email-providers-stacked-comparison/index.tsx#L153-L168)), meaning we could also remove the `context`:

```javascript
{ translate( 'Your password reset email is {{strong}}%(userEmail)s{{/strong}}. {{a}}Change it{{/a}}.', {
```